sources/us/mn/carlton.json

```diff
@@ -9,11 +9,31 @@
         "state": "mn",
         "county": "Carlton"
     },
-    "data": "http://www.carltoncountygis.com/arcgis/rest/services/Map_Layers/MapServer/77",
+    "data": "http://www.carltoncountygis.com/arcgis/rest/services/LINK/Public_LINK_MapLayers/MapServer/7",
     "type": "ESRI",
     "conform": {
-        "number": "Fire_Numbe",
-        "street": "Street_Nam",
-        "type": "geojson"
+        "type": "geojson",
+        "number": {
+            "function": "regexp",
+            "field": "PHYSADDR",
+            "pattern": "^([0-9]+)"
+        },
+        "street": {
+            "function": "regexp",
+            "field": "PHYSADDR",
+            "pattern": "^(?:[0-9]+) (.*)",
+            "replace": "$1"
+        },
+        "city": {
+            "function": "regexp",
+            "field": "PHYSCITY",
+            "pattern": "^(.*) MN$"
```

Comment:
Sorry, I forgot to mention that you'll have to include `"replace": "$1"` in both `city` and `region` to replace then entire value with the first captured group.  It's very close, but the `city` and `region` fields in the output are blank.